files-override/shared/.prettierrc.cjs

```diff
@@ -2,6 +2,7 @@
 
 module.exports = {
   plugins: ['prettier-plugin-ember-template-tag'],
+  singleQuote: true,
```

Comment:
foreshadowing https://github.com/embroider-build/app-blueprint/pull/80/files#r1783208102

Reply:
can't you remove all the other places in this file that change singleQuote to true in the overrides now that this is setting it as the default?

Reply:
Hope so 😅